tests/index.ts

```diff
@@ -239,3 +239,39 @@ test('should format messages.', async t => {
   t.equal(response.payload, '{"jared":"Hello Jared","jaredHanson":"Hello Jared Hanson"}')
   t.equal(response.statusCode, 200)
 })
+
+test('should return error when session is not defined.', async t => {
```

Comment:
This test let me think that should we trigger a startup error instead of a runtime one?

Reply:
Yes, it is better to throw an error in startup as `session` is a must for this plugin.

Reply:
Is it the case to create an issue for?